index.js

```diff
@@ -27,24 +38,25 @@ exports.createApiAction = (apiBaseUrl, method, endpoint) => {
 
         const headers = {
             'Accept': 'application/json',
-            'Content-Type': 'application/json',
             'Authorization': (getState().auth && getState().auth.token) ? "Bearer " + getState().auth.token : ''
         }
 
-        let rsaa = {
+        const body = options.files ? getFormData(options) : JSON.stringify(options.body || '')
```

Comment:
JSON.stringify(options.body || '')

NOT EQUIVALENT TO

options.body ? JSON.stringify(options.body) : ''